deploy/kubernetes/base/controller/cluster_setup.yaml

```diff
@@ -143,6 +146,9 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
Why does this diff show up while the master branch does not really have a diff of this? I saw you r rebasing from [kubernetes-sigs:release-999.2](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/tree/release-999.2) ? Shouldn't it be rebasing onto master?